py/server/deephaven_internal/plugin/object/__init__.py

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+
+import jpy
+
+from typing import Optional, List, Any
+from deephaven.plugin.object_type import Exporter, ObjectType, Reference, MessageStream, FetchOnlyObjectType
+from deephaven._wrapper import JObjectWrapper, wrap_j_object
+
+JReference = jpy.get_type('io.deephaven.plugin.type.ObjectType$Exporter$Reference')
+JExporterAdapter = jpy.get_type('io.deephaven.server.plugin.python.ExporterAdapter')
+JMessageStream = jpy.get_type('io.deephaven.plugin.type.ObjectType$MessageStream')
+
+
+def _adapt_reference(ref: JReference) -> Reference:
+    return Reference(ref.index(), ref.type().orElse(None))
+
+
+def _unwrap(object):
+    # todo: we should have generic unwrapping code ABC
```

Comment:
Does anything need to happen here?  Need a ticket?

Reply:
No, this code is about 18mos old, but github doesn't recognize that the file moved and only had some changes (the closest change here was the newlines/docs/typehints I've added at your request above and below this function).